data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1673,6 +1673,7 @@
     "name": { "str_sp": "parasaurolophus" },
     "copy-from": "mon_hadrosaurus",
     "//": "When large weights and volumes are fixed this should be 4000 L and 4000 kg",
+    "bodytype": "bear",
```

Comment:
```suggestion
```
In theory this shouldn't be needed

Reply:
Yeah you're probably right, I was just a little confused by the discussion in the issue with OP saying that the dinosaur had the elephant body type.